classes/04class/exercises/c04-iac03/nimmi89/README.md

```diff
@@ -0,0 +1,235 @@
+# C04-IAC03
+
+## Terraform plan output
+
+$ terraform plan -var-file ./main.tfvars -var="vpcCIDRblock=172.16.0.0/16" -var='public_subnet_cidr_blocks=["172.16.10.0/24","10.0.11.0/24"]' -var='public_subnet_names=["public-a","renamed-public-b"]' -out vpc_plan
```

Comment:
Better to wrap all of the command/output in \``` , otherwise markdown will mess when rendering

Reply:
Yes!Have done that!